src/ConfigurationContext.tsx

```diff
@@ -73,6 +76,7 @@ const ConfigurationProvider: FunctionComponent = ({ children }) => {
         displayReportAnIssue,
         displaySelfAssessment,
         displaySymptomChecker,
+        findATestCenterUrl,
```

Comment:
This means this URL is not being localized. This is perfectly fine, maybe worth asking if it is supported by the authority or if that's a desire.

Reply:
for now we dont need to support localization of the link here.  We can add it as an enhancement if turns out to be a requirement